test/fixtures/issueClose.json

```diff
@@ -0,0 +1 @@
+{ "id": "hi" }
```

Comment:
i think it would be fine to inline this? it's tiny and only used once. you can do:

```go
http.StubResponse(200, bytes.NewBufferString(`{"id": "hi"}`))
```

Reply:
Very good point...